### DISCUSSION TITLE
How many USB C cable I need to power an ESP Thread Border Router/Zigbee Gateway board ?

### DISCUSSION THREAD

**Discussion by mforbiz** [openthread/openthread#9994] [Q&A] [answered]
Hi.
I've just purchased an ESP Thread Border Router/Zigbee Gateway board + Sub ethernet module.
Anyway I've a doubt.
On the board there are two USB C ports that, I suppose, are needed to power the board + the sub thernet module.
The question is:
must I use two different USB cable connected to two, different, USB ports on the PC or is enough only one USB C cable to power all the system?
![2024-04-04_165834](https://github.com/openthread/openthread/assets/105542757/fc019865-ef79-4fdd-a588-73c0fee72ac0)

**Comment by jwhui** [comment, marked as answer]
You should only need to connect one USB cable. See the picture here:
https://openthread.io/codelabs/esp-openthread-hardware#0
